Locale/zhCN/Locale.xml

```diff
@@ -0,0 +1,3 @@
+<Ui xsi:schemaLocation="http://www.blizzard.com/wow/ui/ ..\FrameXML\UI.xsd">
+  <Script file="config.lua"/>
```

Comment:
Pretty sure this is case-sensitive, however everything else looks awesome so I'm just going to merge it and fix case issue.